content/en/specifications/specification-latest.md

```diff
@@ -0,0 +1,29 @@
+---
+title: Specification index
+weight: 15
+description:
+---
+
+The following doc versions are available:
+
+- [latest](https://theupdateframework.github.io/specification/latest/index.html)
+- [draft](https://theupdateframework.github.io/specification/draft/index.html)
+- [v1.0.33](https://theupdateframework.github.io/specification/v1.0.33/index.html)
+- [v1.0.32](https://theupdateframework.github.io/specification/v1.0.32/index.html)
+- [v1.0.31](https://theupdateframework.github.io/specification/v1.0.31/index.html)
+- [v1.0.30](https://theupdateframework.github.io/specification/v1.0.30/index.html)
+- [v1.0.29](https://theupdateframework.github.io/specification/v1.0.29/index.html)
+- [v1.0.28](https://theupdateframework.github.io/specification/v1.0.28/index.html)
+- [v1.0.27](https://theupdateframework.github.io/specification/v1.0.27/index.html)
+- [v1.0.26](https://theupdateframework.github.io/specification/v1.0.26/index.html)
+- [v1.0.25](https://theupdateframework.github.io/specification/v1.0.25/index.html)
+- [v1.0.24](https://theupdateframework.github.io/specification/v1.0.24/index.html)
+- [v1.0.23](https://theupdateframework.github.io/specification/v1.0.23/index.html)
+- [v1.0.22](https://theupdateframework.github.io/specification/v1.0.22/index.html)
+- [v1.0.20](https://theupdateframework.github.io/specification/v1.0.20/index.html)
+- [v1.0.19](https://theupdateframework.github.io/specification/v1.0.19/index.html)
+- [v1.0.18](https://theupdateframework.github.io/specification/v1.0.18/index.html)
+- [v1.0.17](https://theupdateframework.github.io/specification/v1.0.17/index.html)
+
+Find here an updated list of the
+[spec versions](https://theupdateframework.github.io/specification/)
```

Comment:
Eventually we might not need the link to the GH pages site, but for now I'd like to keep it. Let's adjust the prose though:

```suggestion
This list is also available from
[Spec versions](https://theupdateframework.github.io/specification/).
```